### DISCUSSION TITLE
When is 2.3 scheduled for release?

### DISCUSSION THREAD

**Discussion by transreductionist** [pallets/flask#5016] [Q&A] [answered]
We are reviewing the list of code in 2.2.3 that will be deprecated in 2.3, and are ensuring the changes won't affect our code.

**Comment by davidism** [comment, marked as answer]
We don't keep a strict release cadence. Releases happen when a maintainer has enough time to decide that the release is ready and to make it happen.

**Comment by transreductionist** [comment]
Thanks for the quick response. Keep up the great work! I love Flask.
